src/bsdfs/dielectric.cpp

```diff
@@ -56,9 +56,8 @@ namespace Caramel{
         }
         else{
             const Vector3f local_outgoing_dir = refract(local_incoming_dir, n, in_ior, ex_ior);
-            // https://www.pbr-book.org/4ed/Reflection_Models/Dielectric_BSDF#Non-SymmetricScatteringandRefraction
-            const Float tmp = (ex_ior/in_ior);
-            return {local_outgoing_dir, {tmp, tmp, tmp}, Float0};
+            // TODO : consider https://www.pbr-book.org/4ed/Reflection_Models/Dielectric_BSDF#Non-SymmetricScatteringandRefraction
+            return {local_outgoing_dir, vec3f_one, Float0};
```

Comment:
Need to investigate more... returning eta causes inconsistent result between mitsuba (and it should also return eta^2)